docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+
+As you may have guessed, **each pin has a specific role, and you can use it only for that role**.
+Some of them are input/output, power (3.3V or 5V), or ground.
+As you can see in the diagram below, there are 40 pins on the Pi.
+26 of them are GPIO pins.
+
+<img src="../../img/blink-an-led/pinout.jpg" alt="Diagram showing all of the GPIO pins on a Raspberry Pi 4 and their corresponding pin number and function." width="100%">
+
+One thing that might be confusing is the *board pin numbering* versus *GPIO pin numbering*.
+There are 40 physical pins numbered from 1 to 40.
+That is board pin numbering, corresponding to the pin's physical location on the board.
+Then there's numbering them by function or GPIO connection.
+For example, "GPIO 22".
+**When working with the GPIO pins with Viam, you will use the board pin numbers.**
+For example, board pin 38 is the same pin as GPIO pin 20, but when configuring your robot with Viam, you should refer to it as pin `38`.
+
+Some of the GPIO pins can be used not only as GPIO pins but for other more specific functions as well.
+The website [pinout.xyz](https://pinout.xyz/) is a helpful resource with the exact layout and role of each pin.
+
+### Wire your circuit
+
+{{% alert title="Caution" color="caution" %}}
+Disconnect your Raspberry Pi and any other electronics from power before touching or moving wires on them.
+{{% /alert %}}
+
+Now it's time to wire your circuit according to the diagram below.
+It doesn't matter what color jumper wires you use but for reference, the blue wire below connects the LED to ground, and the red wire connects the resistor to pin 8 (GPIO 14).
+
+<img src="../../img/blink-an-led/circuit.png" alt="Circuit diagram showing a Raspberry Pi with a red connector running out of GPIO pin 8 to a 100-ohm resistor. The resistor is connected to the long lead of a red LED bulb. Finally, a blue connector connects the short lead of the LED to the ground connection on pin 6 of the Raspberry Pi GPIO pins." width="100%">
+
+The resistor and LED need to be in series as in the diagram above.
+To find the right resistor use the resistor color code -- for a 100 ohm resistor, it needs to be brown-black-brown.
+You can use your multimeter to double-check the resistor value or check yours using the photo below.
+
+<img src="../../img/blink-an-led/resistor.jpg" alt="Photo of a 100-ohm resistor with text overlaid that says, in order, brown-black-brown-gold." width="50%">
+
+When hooking up the circuit, **note the *polarity* of the LED**.
+You will notice that the LED has long and short leads.
+The long lead is the positive side, which is known as the anode.
+The short lead is the negative side, which is known as the cathode.
+Connect the long anode to the resistor and the short cathode to ground (pin 6) on the Raspberry Pi.
+
```

Comment:
I think we're missing a step where we tell readers to turn the power back on

Reply:
Done!